qa/rpc-tests/test_framework/util.py

```diff
@@ -1152,7 +1152,7 @@ def findBitcoind():
 def standardFlags():
     flags = [] # ["--nocleanup", "--noshutdown"]
     if os.path.isdir("/ramdisk/test"):
-        flags.append("--tmppfx=/ramdisk/test")
+        flags.append("--tmpdir=/ramdisk/test/t")
```

Comment:
?

Reply:
tmppfx used to have the semantics of tmpdir, but we switched those semantics and created the "tmpdir" variable to have the old semantics without updating the use here.  Basically, when you are running tests for debugging, you want the test directory to be well known.  That way to load up the logs, run bitcoin-cli with --datadir, etc, you don't have to determine the PID of the process to determine the temporary directory and then retype the command.  Instead, since the directory is always the same, a few up arrows suffice.

Reply:
thanks for the explanation